_includes/keynotes.html

```diff
@@ -0,0 +1,27 @@
+
+<h1>Keynotes</h1>
+
+<h2><a name="keynote1">Concurrent Data Sketches</a>, Idit Keidar</h2>
```

Comment:
I'd add the date here too, just to be extra clear.
The table with the program is a bit far, and people may have to scroll back and forth when they are skimming the keynote abstracts and the dates.

Reply:
ack. done